classes/local/store/object_file_system.php

```diff
@@ -316,6 +321,50 @@ public function delete_object_from_local_by_hash($contenthash, $objectsize = 0)
         return $finallocation;
     }
 
+    /**
```

Comment:
missing php doc which explains what is going on here

Reply:
doc updated